### PR TITLE
Build: Update to Babel 6

### LINKED ISSUE #37: Update to Babel 6
Several dependencies of js-reporters are outdated. Like babel 5, it's no longer maintained, and changed configuration in version 6.

I suppose the deps should be updated. And I'm going to fix this.

### PR DESCRIPTION
**Pull request by frantic1048**
- uptate Babel to 6
- update ESLint to 2.2 to work with Babel 6

Fixes #37

### CODE REVIEW

**Comment by jzaefferer** [comment]
Travis build failed, looks legit. Please comment here when you push more commits (there are no notifications for pushes).

**Comment by frantic1048** [comment]
Node run into global modules on my machine, causes misconception to the dependencies. I should restrict my global modules..

File: package.json [hunk 1]
```diff
@@ -1,22 +1,30 @@
 {
   "name": "js-reporters",
-  "version": "1.0.0",
+  "version": "1.0.3",
```
Comment (jzaefferer) [inline package.json]: Let's hold off on changing the version until we actually do a release.

**Comment by jzaefferer** [comment]
Besides the version change, this looks good.

File: package.json (hunk 1 above)
Comment (frantic1048) [inline package.json]: All right, I get it~

**Comment by jzaefferer** [comment]
Landed, thank you!

**Pull request merged**